### DISCUSSION TITLE
github action Deploying vuepress to github.io does not display CSS and has wrong image paths.

### DISCUSSION THREAD

**Discussion by ykla** [vuepress/core#981] [Q&A]
Hi,

vuepress deploying to GitHub pages does not show css and images. I looked at the GitHub action generated website source code, which is linked to .gitbook, but the generated static website pages do not have this folder.

https://github.com/ykla/ykla.github.io
https://ykla.github.io/

Thanks.

**Comment by Mister-Hope** [comment]
# Detailed information is required, e.g.: a wrong page and its source code.

Also, I don't think https://ykla.github.io/ is a vuepress site. Try at least learning some basics, instead of showing an apple and told us this "banana" taste bad
